src/query/stream_to_xasl.c

```diff
@@ -2413,6 +2418,26 @@ stx_build_cte_xasl_id (THREAD_ENTRY * thread_p, char *ptr, XASL_ID * cte_xasl_id
   return ptr;
 }
 
+static SQ_CACHE *
+stx_restore_sq_cache (THREAD_ENTRY * thread_p, char *ptr, int n_elements)
```

Comment:
I think the `stx_build_` prefix should be used.
Functions using the 'stx_restore_' prefix call the stx_get_struct_visited_ptr and stx_mark_struct_visited functions.

Reply:
I've applied your review. Please confirm.